app-games/src/main/java/com/aptoide/android/aptoidegames/theme/AppGamesButton.kt

```diff
@@ -293,7 +297,11 @@ fun AppGamesOutlinedButton(
       )
     ),
     colors = ButtonDefaults.buttonColors(
-      backgroundColor = Color.Transparent,
+      backgroundColor = if (style is Default) {
+        AppTheme.colors.primary.copy(alpha = 0.2f)
```

Comment:
This button background should be transparent. The colored button with this alpha effect should only appear when a click action happens.